docs/management/admin/event-filters.asciidoc

```diff
@@ -25,24 +27,33 @@ In the KQL search bar, enter the following query: `event.dataset : endpoint.even
 [role="screenshot"]
 image::images/event-filter.png[]
 . Enter a name for the event filter.
-. Depending on which page you add the filter, either modify the pre-populated conditions or add new conditions that define when {elastic-sec} filters events. You can define multiple conditions with `AND` relationships. You can also add nested conditions. In the image above, the event filter excludes events whose `event.category` field is `network`, and whose `process.executable` field is the same as the specified path.
-. Add a new comment that describes or identifies the filter (optional).
-. Click *Add event filter*. The new filter is added to the Event filters list.
+. Enter a filter description (optional).
+. Depending which page you are using to create the filter, either modify the pre-populated conditions or add new conditions to define when {elastic-sec} filters events. You can define multiple conditions with `AND` relationships. You can also add nested conditions. For example, the event filter pictured above excludes events whose `event.category` field is `network`, and whose `process.executable` field is as specified.
+. Select an option in the *Assignment* section to assign the event filter to a specific integration policy:
++
+* `Global`: Assign the event filter to all integration policies for {endpoint-sec}.
+* `Per Policy` (Platinum or Enterprise subscription only): Assign the event filter to one or more specific {endpoint-sec} integration policies. Select each policy in which you want the events to be filtered.
++
+NOTE: You can also select the `Per Policy` option without immediately assigning a policy to the event filter. For example, you could do this to create and review your event filter configurations before putting them into action with a policy.
```

Comment:
```suggestion
NOTE: You can also select the `Per Policy` option without immediately assigning a policy to the event filter. For example, you could do this to create and review your event filter configurations before activating them with a policy.
```

Reply:
Not sure if this is how it works, but offering the suggestion nevertheless in case it is.

Reply:
Ah ok, I see what you're saying. In that case, maybe "enacting"? Also, the original works as is, so whatever you choose!